homeassistant/components/overseerr/event.py

```diff
@@ -38,6 +38,17 @@ class OverseerrEventEntityDescription(EventEntityDescription):
         ],
         nullable_fields=["comment", "issue"],
     ),
+    OverseerrEventEntityDescription(
+        key="issue",
+        translation_key="last_issue_event",
+        event_types=[
+            "reported",
+            "comment",
```

Comment:
```suggestion
            "commented",
```
I kinda like the idea of having the states answer the question "what happened?" "somebody <action>". But I can also see that it makes the code uglier, so I am fine with it as it is, just wanted to write down my thought

Reply:
Oh I see what you mean

Reply:
#139263

Reply:
I just did some tests and the webhook returns the following:
```
{'notification_type': 'ISSUE_COMMENT', 'subject': "xxx", 'message': 'Ended early :(', 'media': {'media_type': 'tv', 'tmdb_id': xxx, 'tvdb_id': xxx, 'status': 'partially_available', 'status4k': 'unknown'}, 'request': None, 'issue': {'issue_id': 12, 'issue_type': 'VIDEO', 'issue_status': 'OPEN', 'reported_by_email': 'xxx', 'reported_by_username': 'xxx', 'reported_by_avatar': 'xxx', 'reported_by_settings_discord_id': '', 'reported_by_settings_telegram_chat_id': ''}}
```

So unfortunately we need to use `comment` :(

Reply:
something like this? (last commit)